modules/services/mapilio.js

```diff
@@ -348,25 +362,39 @@ export default {
         if (!d) return this;
 
         let wrap = context.container().select('.photoviewer .mapilio-wrapper');
-        let attribution = wrap.selectAll('.photo-attribution').text('');
-
-        if (d.capture_time) {
+        let attribution = wrap.selectAll('.photo-attribution').text('\u00A0');
+      
+        getUserData(d.created_by_id).then((username) => {
```

Comment:
If the username fails to load, everything else in the footer will be hidden too. 

I think this could be optimised by appending the DOM element during the first render, but leaving the element empty until the API response is received.